README.md

```diff
@@ -27,7 +27,7 @@ Search for "Open Policy Agent" in the Extensions (Shift ⌘ X) panel and then in
 
 | Field | Default | Description |
 | --- | --- | --- |
-| `opa.path` | `null` | Set path of OPA executable. |
+| `opa.path` | `null` | Set path of OPA executable. The variable `${workspaceFolder}` will be resolved as the current workspace root. |
```

Comment:
When I first read this I was a a bit confused and didn't understand the significance of the second sentence. Can we make it a bit more explicit. Something like:

```
Set path of OPA executable. If the paths contains the string `${workspaceFolder}`
it will be replaced with the current workspace root. For example,
if the path is set to `${workspaceFolder}/bin/opa` and the current workspace root
is `/home/alice/project`, the OPA executable path will resolve to `/home/alice/project/bin/opa`. 
```